apps/ledger-live-mobile/src/components/RequiresCameraPermissions/Fallback.tsx

```diff
@@ -27,6 +26,9 @@ const FallbackCameraBody: React.FC<Props> = ({
 }: Props) => {
   const { colors } = useTheme();
 
+  // benefit from component scope colors variable for IconSettings
+  const IconSettings = () => <Icon name="settings" size={16} color={colors.constant.white} />
```

Comment:
On dark mode, the icon should be black as the background is white, I think that we have some colors for that in the theme, could you find the one we need, please?

Reply:
good point,  i will figure it out

Reply:
@KVNLS in the dark mode (i used the ForceTheme container over the Fallback) the button's background is still white , so making the icon white will make it invisible. I think i will use the `neutral.c100` color used initially for this theme mode. This will be the result 
![image](https://github.com/user-attachments/assets/b418f20d-7e95-480d-8165-55e3a73fd030)

Reply:
@KVNLS in the dark mode (i used the ForceTheme container over the Fallback) the button's background is still white , so making the icon white will make it invisible.

Reply:
you can use colors.neutral.c00 or colors.neutral.c100 :) It will works for both Theme

Reply:
it will be like this in the dark mode
![image](https://github.com/user-attachments/assets/7ef56e9e-0c33-49c7-b373-2147bafa1a64)

Reply:
But you didn't changed const color ?